definitions/npm/react-with-separator_v1.x.x/flow_v0.72.x-/react-with-separator_v1.x.x.js

```diff
@@ -0,0 +1,10 @@
+declare module "react-with-separator" {
+  import type { ChildrenArray } from "react";
+
+  declare module.exports: React$StatelessFunctionalComponent<{
+    children?: React$Node,
+    leading?: boolean,
+    separator: ChildrenArray<string | number | React$Element<any>>,
```

Comment:
I think `separator: React$ElementType` is enough here, given the lib source code.

Reply:
Great point!

It looks like it is typed to not accept `string` either - https://github.com/facebook/flow/blob/41b0eab99cdc5199421f7cccad9e0c4950f8b2f9/tests/react_custom_funs/clone_element.js#L23

Reply:
But this case is handled manually - https://github.com/LinusU/react-with-separator/blob/0027cc509653a825cfc77e8963e17fef05acb50f/index.js#L6